elasticsearch/estest/client.go

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package test
+package estest
```

Comment:
It was an easy way to avoid an unneeded import alias (eg https://github.com/elastic/apm-server/pull/2974/commits/f26d6150f9912e26db2fdcc8fa705310c4cf0030#diff-2b990608de1ddbd3f6ec60d8b0e68f25R33)
It also seems to be conventional (eg. see `httptest` inside `http` I assume for the same reason  https://golang.org/pkg/net/http/httptest/), but I can revert if you feel strong about it.

I pushed changes for the other comments